server/utils/apicalls.js

```diff
@@ -1,5 +1,19 @@
 var request = require('request-promise-native'); // Request library with native JS promises
-var dotenv = require('dotenv').config();
+var dotenv = require('dotenv').config();  // includes api keys
+
+// google maps js api loader
+module.exports.googleMapsLoader = function() {
+  let url = 'https://maps.googleapis.com/maps/api/js'
+
+  // build query string for api call
+  let qs = {
+    key: dotenv.GOOGLE_PLACE_API_KEY,
```

Comment:
GOOGLE_PLACE_API_KEY does not exist, we DO have GOOGLE_PLACE_S__API_KEY, but wouldn't we want GOOGLE_MAPS_API_KEY? Could be the cause of the API key error.